client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -204,6 +204,13 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 		} as React.CSSProperties;
 	}
 
+	const RedirectToFirstStep = () => {
+		window.location.replace(
```

Comment:
If we wanted to keep using `react-router-dom`, you can use `useNavigate` with `replace: true`. [Docs.](https://reactrouter.com/en/main/hooks/use-navigate)

Reply:
Might make handling tests easier, too.